FabricObserver/Observers/AppObserver.cs

```diff
@@ -1784,6 +1878,40 @@ private Task<ParallelLoopResult> MonitorDeployedAppsAsync(CancellationToken toke
                         checkMemPct = true;
                     }
 
+                    // Memory - Private Bytes MB.
+                    if (application.ErrorPrivateBytesMb > 0 || application.WarningPrivateBytesMb > 0 || MonitorResourceGovernanceLimits)
+                    {
+                        _ = AllAppPrivateBytesDataMb.TryAdd(id, new FabricResourceUsageData<float>(ErrorWarningProperty.PrivateBytesMb, id, 1, false, EnableConcurrentMonitoring));
+                    }
+
+                    if (AllAppPrivateBytesDataMb.ContainsKey(id))
+                    {
+                        checkMemPrivateBytes = true;
+                    }
+
+                    // Memory - RG monitoring.
+                    if (MonitorResourceGovernanceLimits && repOrInst.RGEnabled && repOrInst.RGMemoryLimitMb > 0)
+                    {
+                        _ = AllAppRGMemoryUsagePercent.TryAdd(id, new FabricResourceUsageData<double>(ErrorWarningProperty.RGMemoryUsagePercent, id, 1, false, EnableConcurrentMonitoring));
+                    }
+
+                    if (AllAppRGMemoryUsagePercent.ContainsKey(id))
+                    {
+                        rgMemoryPercentThreshold = application.WarningRGMemoryLimitPercent;
+
+                        if (rgMemoryPercentThreshold > 0)
+                        {
+                            if (rgMemoryPercentThreshold < 1)
+                            {
+                                rgMemoryPercentThreshold = application.WarningRGMemoryLimitPercent / 100; // decimal to double.
+                            }
+                        }
+                        else
+                        {
+                            rgMemoryPercentThreshold = 90; // Default: 90%.
```

Comment:
minor: this value is also used at the line 455 - might be useful to have defined constant parameter for it.